src/cljs-lib/src/calva/repl/webview/core.cljs

```diff
@@ -156,18 +163,28 @@
   [^js webview-panel]
   (.. webview-panel (onDidDispose (fn [] (dispose-repl-output-webview-panel repl-output-webview-panel)))))
 
+(defn handle-message
+  [message]
+  (let [message-data (reader/read-string message)
+        command-name (:command/name message-data)]
+    (case command-name
+      "save-state" (save-state message-data))))
+
 (defn initialize-webview-panel
-  [context ^js webview-panel ^js state]
+  [context ^js webview-panel state]
+  (.. webview-panel -webview (onDidReceiveMessage handle-message))
   (add-listeners! webview-panel)
   (add-subscriptions! context {:webview-panel webview-panel})
-  (if (and state (.-html state))
-    (set! (.. webview-panel -webview -html) (.-html state))
+  (if (and state (:html state))
+    ;; TODO: Address console warnings and errors when this code is run
+    (set! (.. webview-panel -webview -html) (:html state))
```

Comment:
Leaving this TODO intentionally. I'm not sure how to fix the errors and warnings, but Calva functions as intended anyway, so I don't feel it's worth spending a lot of time on right now. I like to keep the console clean, but I also don't like to spend much time on that if the warnings and errors aren't related to an actual functionality problem and they're infrequent (only happen when the output view is reopened / reloaded).